proposals/memory-specializations/intro.tex

```diff
@@ -0,0 +1,72 @@
+\rSec0[intro]{General}
+\rSec1[intro.scope]{Scope}
+
+\pnum
+This proposal describes functions compatible with the Ranges Technical
+Specification~(\ref{intro.refs}) that shall not be ratified prior to the
+publication of the TS. These extensions change and add to the existing
+memory specialisation library facilities found in N4606~(\ref{intro.refs}).
+
+\rSec1[intro.refs]{References}
+This document requires some familiarity with the following documents.
+
+\begin{itemize}
+\item JTC1/SC22/WG21 N4606,   \doccite{Working Draft, Standard for Programming Language \Cpp}
+\item ISO/IEC TS 19217:2015,  \doccite{Programming Languages - \Cpp Extensions for Concepts}
+\item JTC1/SC22/WG21 P0459R0, \doccite{\Cpp Extensions for Ranges, Speculative Combined Proposal Document}
```

Comment:
If the TS working draft wasn't so hopelessly out of date, I would suggest referencing it. P0459 seems like a decent compromise for the time being. Ideally we will have a new TS working draft to reference before it makes a difference.